.github/workflows/integration.yml

```diff
@@ -21,19 +18,18 @@ jobs:
       NODE_VERSION: "lts/*"
 
     steps:
-    - name: Checkout repository (pull request)
+    - name: Checkout repository
       uses: actions/checkout@v2
-      if: ${{ github.event_name == 'pull_request_target' }}
       with:
         fetch-depth: 0
-        ref: ${{ github.event.pull_request.head.ref }}
-        repository: ${{ github.event.pull_request.head.repo.full_name }}
 
-    - name: Checkout repository (push)
-      uses: actions/checkout@v2
-      if: ${{ github.event_name == 'push' }}
-      with:
-        fetch-depth: 0
+    - name: Update PR checklist for surge.sh as pending (pull request)
+      if: ${{ github.event_name == 'pull_request' }}
+      env:
```

Comment:
Can't it be a global env for simplicity?

Reply:
The variables are only available for pull requests only, so it is not possible to have it set globally for all steps.

Reply:
If you are using a condition repetitively, you can create a job to store its outputs globally and then use that output for the rest of the workflows, but this kind of abstraction can make the code a bit difficult to understand, so not sure if it is worth it.